proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -76,8 +76,10 @@ message Validator {
   option (gogoproto.goproto_getters)  = false;
 
   string              operator_address = 1 [(gogoproto.moretags) = "yaml:\"operator_address\""];
-  google.protobuf.Any consensus_pubkey = 2
-      [(cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey", (gogoproto.moretags) = "yaml:\"consensus_pubkey\""];
+  google.protobuf.Any consensus_pubkey = 2 [
+    (cosmos_proto.accepts_interface) = "cosmos.crypto.v1beta1.PubKey",
```

Comment:
what does this tag do? Would it be beneficial to add it to the solo machine `Any`s?

Reply:
Ideally the solo machine can take in as an `Any` any public key that fulfills the public key interface and is registered on the codec (can be done in app.go)

Reply:
I think right now it does exactly nothing (correct me if i'm wrong @aaronc), but ultimately it should remove the `.RegisterInterface` and `.RegisterImplementation` calls we do in modules.

Also see https://github.com/regen-network/cosmos-proto#accepts_interface